src/main/java/ai/rev/speechtotext/models/NlpModel.java

```diff
@@ -0,0 +1,33 @@
+package ai.rev.speechtotext.models;
+
+import com.google.gson.annotations.SerializedName;
+
+/** Model type. */
+public enum NlpModel {
+
+    /** gpt 3.5. This is the default value. */
+    @SerializedName("standard")
+    STANDARD("standard"),
+
+    /** gpt 4. */
+    @SerializedName("premium")
+    PREMIUM("premium");
+
+    private final String model;
+
+    NlpModel(String model) {
+        this.model = model;
+    }
+
+    /**
+     * Returns the String value of the enumeration.
+     *
+     * @return The String value of the enumeration.
+     */
+    public String getModel() { return model; }
+
+    @Override
+    public String toString() {
+        return "{" + "model='" + model + '\'' + '}';
```

Comment:
what is the purpose of having it in this format? why can't consumer of this type format it as needed?

Reply:
I don't know what the intention was, but I follow the same pattern. JobOptions class at the top level calls .toString() on every member and expects this output.

Reply:
yeah, really. at last for some nested models we have this formatting.
@amikofalvy by any chance do you know what was the purpose?